.github/ISSUE_TEMPLATE/fuzzer.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Fuzzer Report
+description: Report an issue with the fuzzer or found through fuzzing.
+labels: ["bug", "fuzzer-found", "fuzzer"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please add as much relevant information to the form below as you can.
+        This will help us replicate and solve the issue. Thank you!
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please describe the issue.
+      placeholder: "[Expected behavior] and [actual behavior]."
+    validations:
+      required: true
+  - type: textarea
+    id: seed
+    attributes:
+      label: Seed 
+      placeholder: Fuzzer Seed
```

Comment:
The seed by itself may not guarantee the reproduction of the error, because some other things also affect the random generation of expression and data, such as the base commit, the skip function list, the --only command line argument, etc. Maybe replace "seed" by "error reproduction".

Reply:
Its an optional feed. I will edit it to say Optional.

Reply:
Sure, what about "Error reproduction parameters"?

Reply:
@kgpai  missed your comment. Go ahead!

Reply:
Fuzzer invocation arguments , is a bit more obvious imv.